File: projeto_2 - Kaella Sales/src/Carro.cpp
#include "../include/Carro.h"
#include <iostream>

// Construtor da classe Carro
Carro::Carro(int a, std::string m, std::string mo, std::string c, std::string p) {
    ano = a;
    marca = m;
    modelo = mo;
    cor = c;
    placa = p;
}

// Getter para o modelo
std::string Carro::getModelo() const {
    return modelo;
}

// Getter para o ano
int Carro::getAno() const {
    return ano;
}

//Getter para a marca
std::string Carro::getMarca() const {
    return marca;
}


// Método para mostrar as informações do carro
void Carro::exibirDetalhes() {
    std::cout << "Carro: " << marca << " " << modelo << " (" << ano << ")\n";
    std::cout << "Cor: " << cor << " | Placa: " << placa << "\n" << std::endl;
}


void Carro::setAtualizarCor(std::string novaCor) {
    cor = novaCor;
}

File: carro.py
class Carro:
    def __init__(self, ano, marca, modelo, cor, placa):
        self.ano = ano
        self.marca = marca
        self.modelo = modelo
        self.cor = cor
        self.placa = placa

    def exibir_detalhes(self):
        print(self.marca, self.modelo, self.ano, self.cor, self.placa)

    def atualizar_cor(self, nova_cor):
        self.cor = nova_cor
